hack/release.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Renders and copies documentation files into the informed RELEASE_DIR, the script search for
+# task templates on a specific glob expression. The templates are rendered using the actual
+# task name and documentation is searched for and copied over to the task release directory.
+#
+
+shopt -s inherit_errexit
+set -eu -o pipefail
+
+readonly RELEASE_DIR="${1:-}"
+
+# Print error message and exit non-successfully.
+panic() {
+    echo "# ERROR: ${*}"
+    exit 1
+}
+
+# Extracts the filename only, without path or extension.
+extract_name() {
+    declare filename=$(basename -- "${1}")
+    declare extension="${filename##*.}"
+    echo "${filename%.*}"
+}
+
+# Finds the respective documentation for the task name, however, for s2i it only consider the
+# "task-s2i" part instead of the whole name.
+find_doc() {
+    declare task_name="${1}"
+    find docs/ -name "${task_name}*.md"
+}
+
+#
+# Main
+#
+
+release() {
+    # making sure the release directory exists, this script should only create releative
+    # directories using it as root
+    [[ ! -d "${RELEASE_DIR}" ]] &&
+        panic "Release dir is not found '${RELEASE_DIR}'!"
+
+    # releasing all task templates using the following glob expression
+    declare task_name=task-buildpacks
+    declare task_doc=README.md
```

Comment:
Today we have only one task in this repo so this way it works
But in future what if more tasks gets added (Not sure)  then this script won't work isn't it ?

Reply:
Yes @savitaashture then we need to modify the code similar to `task-containers` repository.

Reply:
yep we do it in that way only, because its not sure that this repo will definitely get more tasks , it could be a repo for only this task, so then changes to the script wouldnt make sense then. so we usually have these differences in script.

Reply:
okay make sense to me